app/config/schema.json

```diff
@@ -809,6 +809,34 @@
       },
       "additionalProperties": false,
       "required": [ "handlers" ]
+    },
+    "smtp": {
+      "type": "object",
+      "title": "SMTP configuration",
+      "properties": {
+        "host": {
+          "type": "string",
+          "title": "SMTP host",
+          "default": "localhost"
+        },
+        "username": {
+          "type": "string",
+          "title": "SMTP username",
+          "minLength": 8,
+          "default": "NOT A USERNAME"
+        },
+        "password": {
+          "type": "string",
+          "title": "SMTP password",
+          "minLength": 8,
+          "default": "NOT A PASSWORD"
```

Comment:
strictly speaking we don't have a default value here and the text might be confusing, but that's not a blocker.

Reply:
Just removed them